path/fount.ps1

```
@@ -751,65 +782,101 @@ function fount_upgrade {
-			}
+		Write-Warning (Get-I18n -key 'git.noUpstreamBranch' -params @{branch = $currentBranch })
+		Invoke-GitForFount branch --set-upstream-to origin/master
+		$remoteBranch = 'origin/master'
+	}
+
+	if (-not (Test-FountGitRef $remoteBranch)) {
+		Write-Warning (Get-I18n -key 'git.remoteRefUnavailable' -params @{ ref = $remoteBranch })
+		return
+	}
+
+	$mergeBase = Invoke-GitForFount merge-base $currentBranch $remoteBranch 2>$null
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Invoke-GitForFount' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)